apis/v1beta1/allocation_strategy.go

```diff
@@ -30,6 +30,9 @@ const (
 	// TargetAllocatorAllocationStrategyConsistentHashing targets will be consistently added to collectors, which allows a high-availability setup.
 	TargetAllocatorAllocationStrategyConsistentHashing TargetAllocatorAllocationStrategy = "consistent-hashing"
 
+	// TargetAllocatorAllocationStrategyConsistentHashingPerNode targets will be assigned to the collector on the node they reside on (use only with daemon set).
+	OpenTelemetryTargetAllocatorAllocationStrategyPerNode TargetAllocatorAllocationStrategy = "per-node"
```

Comment:
It would be good to align the name with the other ones e.g. `TargetAllocatorAllocationStrategyPerNode`